### PR TITLE
Smaller sns footprint

### PR DESCRIPTION
**Pull request by jruts**
fixes https://github.com/numo-labs/lambda-tile-provider/issues/20

### CODE REVIEW

File: lib/articleFormatter.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  name: item.name,
+  sections: item.sections,
+  url: item.url
+});
+
+const sns = (item) => ({
+  id: item.id,
+  type: 'article',
+  name: item.name,
+  sections: item.sections && item.sections.length > 0 ? [{image: item.sections[0].image}] : []
```
Comment (jruts) [inline lib/articleFormatter.js]: The reason I keep this structure is so the UI will not break.

File: test/articleFormatter.test.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+const expect = require('chai').expect;
+const formatter = require('../lib/articleFormatter');
+
+const item = {
+  id: '42',
+  name: 'Pink Fluffy Unicorns',
```
Comment (nelsonic) [inline test/articleFormatter.test.js]: you couldn't resist... 😜 
https://www.youtube.com/watch?v=a-xWhG4UU_Y

File: test/contentHandler.test.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 require('env2')('.env');
-const assert = require('assert');
+const expect = require('chai').expect;
```
Comment (nelsonic) [inline test/contentHandler.test.js]: ask @hassy about the interesting issue he found with `chai` last week... 😉

**Comment by nelsonic** [comment]
looks good @jruts 👍 
merging. 🚀

**Pull request merged**